tests/unit/ForeignJurisdiction.spec.ts

```diff
@@ -0,0 +1,104 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { mount } from '@vue/test-utils'
+import { ForeignJurisdiction } from '@/components/common'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+
+describe('ForeignJurisdiction', () => {
+  it('initializes correctly', () => {
+    const wrapper = mount(ForeignJurisdiction,
+      {
+        vuetify
+      })
+    const vm: any = wrapper.vm
+
+    expect(vm.selectedCountry).toBe('')
+    expect(vm.selectedRegion).toBe('')
+    expect(vm.countryNames.length).toBe(249)
```

Comment:
This may change over time since we're importing a package and the world is changing.

However, I say leave it. Some future developer may have to update this but it serves the purpose of verifying that the country array is loaded.

Add a comment that this may change, or similar.